### ISSUE TITLE
Problem by reproduce the CT Model

### ISSUE THREAD

**Issue by SvenLang** [sunlab-osu/TURL#18]
I want to fine-tune the pre-trained model to the column type model based on your provided code and data. The finetuning with the fine_tune_CT.sh runs without problems but when I then validate the output model in evaluate_task.ipynb I have a F1-Score of 0.07. What I did was the following:

1.  downloaded data & pretrained model provided by your link
2.  downloaded bert-base-uncased from https://huggingface.co/bert-base-uncased and copied into data/pre-trained_models/bert-base-uncased
3.  run the fine_tune_CT.sh script
4.  evaluate the model in evaluate_task.ipynb

As I mentioned above the F1-Score is very low. When I evaluate your provided already fine-tuned colum_type model, i get to the reported scores in your paper. But not with the own fine-tuned model. Am I doing something wrong?

**Comment by xiang-deng** [comment]
Hi sorry for the confusion. Our own dataset is a multi-label classification and should BCE loss instead of the CrossEntropy loss default in the code base. CrossEntropy loss should be used for datasets where each column only has a single type. I have changed that and updated the instruction. Thanks for reporting the issue.

**Comment by A11en0** [comment]
The instruction metioned Line 1033, but it's rather CT task(HybridTableCT) than TR(HybridTableTR) task, it could be a typo?
